src/scss/components/_modal-waitlist.scss

```diff
@@ -0,0 +1,30 @@
+.waitlist-modal {
+  .close {
+    font-size: $font-size-xs;
+    color: $sh-purple;
+    opacity: 1;
+
+    &:hover {
+      color: $sh-purple;
+    }
+  }
+
+  .masking-select {
+    color: $custom-select-color;
+    padding: $custom-select-padding-y ($custom-select-padding-x + $custom-select-indicator-padding) $custom-select-padding-y $custom-select-padding-x/1.5;
+    border: $custom-select-border-width solid $custom-select-border-color;
+    background: $custom-select-bg $custom-select-background-mask;
+    width: calc(100% - #{$grid-gutter-width});
+
+    &.selected {
+      color: $custom-select-color;
+    }
+
+    @include border-radius($custom-select-border-radius, 0);
```

Comment:
for border-radius use the default utility class directly on the DOM

Reply:
updated with rounded class and remove from waitlist